src/server/common/model/state/state-manager.test.js

```diff
@@ -110,3 +110,86 @@ describe('StateManager.key', () => {
     )
   })
 })
+
+describe('StateManager.setSection', () => {
+  let mockSectionV2
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+    mockSectionV2 = /** @type {any} */ ({
```

Comment:
This could be a `const` I think, and is more test data than a mock?

(again, just a cleanup comment)